_translator-files/po/de/Privacy-Statement.po

```diff
@@ -30,110 +32,110 @@ msgstr "wiki"
 #: ../wiki/en/Privacy-Statement.md:1
 #, no-wrap
 msgid "/wiki/Privacy-Statement"
-msgstr ""
+msgstr "/wiki/Privacy-Statement"
 
 #. type: YAML Front Matter: title
 #: ../wiki/en/Privacy-Statement.md:1
 #, no-wrap
 msgid "Jamulus – Privacy Statement"
-msgstr ""
+msgstr "Jamulus - Erklärung zum Datenschutz - (Die englische Version ist massgeblich.)"
 
 #. type: Plain text
 #: ../wiki/en/Privacy-Statement.md:9
 msgid "# Privacy Statement"
-msgstr ""
+msgstr "# Erklärung zum Datenschutz - (Die englische Version ist massgeblich.)"
 
 #. type: Plain text
 #: ../wiki/en/Privacy-Statement.md:11
 msgid "## Definition of Terms"
-msgstr ""
+msgstr "## Definition der Begriffe"
 
 #. type: Plain text
 #: ../wiki/en/Privacy-Statement.md:13
 msgid "\"**Server**\" The Jamulus Server software, as opposed to the host machine/OS it is running on."
-msgstr ""
+msgstr "\"**Server**\" Die Jamulus Server-Software, im Gegensatz zum Host-Rechner/OS, auf dem sie läuft."
 
 #. type: Plain text
 #: ../wiki/en/Privacy-Statement.md:14
 msgid "\"**Client**\" The Jamulus software used to connect to a Server"
-msgstr ""
+msgstr "\"**Client**\" Die Jamulus-Software, die zur Verbindung mit einem Server verwendet wird"
 
 #. type: Plain text
 #: ../wiki/en/Privacy-Statement.md:15
 msgid "\"**Directory**\" A Jamulus Server configured to supply a list of Servers to Clients"
-msgstr ""
+msgstr "\"**Verzeichnis**\" Ein Jamulus-Server, der so konfiguriert ist, dass er den Clients eine Liste von Servern zur Verfügung stellt"
 
 #. type: Plain text
 #: ../wiki/en/Privacy-Statement.md:17
 msgid "## Web site"
-msgstr ""
+msgstr "## Web Seite"
```

Comment:
Web Seite --> Website

Just for clarification how I use Webseite vs Website:

Webseite (webpage) = the .html page e.g. https://jamulus.io/wiki/index.html
Website = the whole website with all sub sites. I.e: all sites behind jamulus.io

Reply:
Changed to "Website"